libs/components/packages/src/schematics/migrations/update-10/axe-core/axe-core.schematic.ts

```diff
@@ -13,16 +19,19 @@ import {
 export default function updateAxeCore(): Rule {
   return (tree, context) => {
     if (getPackageJsonDependency(tree, '@skyux-sdk/testing') !== null) {
-      const axeCore = getPackageJsonDependency(tree, 'axe-core');
-
-      if (axeCore !== null) {
-        axeCore.version = '~4.8.3';
-        axeCore.overwrite = true;
+      const axeCore: NodeDependency = {
+        name: AXE_CORE_PACKAGE,
+        version: AXE_CORE_VERSION,
+        type: NodeDependencyType.Dev,
+        overwrite: true,
+      };
 
-        addPackageJsonDependency(tree, axeCore, 'package.json');
+      const packageJsonPath = '/package.json';
 
-        context.addTask(new NodePackageInstallTask());
-      }
+      removePackageJsonDependency(tree, AXE_CORE_PACKAGE, packageJsonPath);
+      addPackageJsonDependency(tree, axeCore, packageJsonPath);
     }
+
+    context.addTask(new NodePackageInstallTask());
```

Comment:
Do we still need to add this task if we don't have anything to update?

Reply:
Yeah, in the cases where `@skyux-sdk/testing` is installed but `axe-core` is not (we install `axe-core` in this schematic if it's not installed already).